### ISSUE TITLE
The page isn’t redirecting properly - socialturn/connect/facebook?code=

### ISSUE THREAD

**Issue by alexionegit** [anantgarg/socialturn#13]
Hello,

When I press the "Connect" button of my socialturn account I get a nasty error. Please note that my cookes are enabled.

The page isn’t redirecting properly

Firefox has detected that the server is redirecting the request for this address in a way that will never complete. This problem can sometimes be caused by disabling or refusing to accept cookies.

Could you please advise?
Many thanks!

Alex

![screen capture](https://user-images.githubusercontent.com/29202416/31088783-8f6da8a6-a7aa-11e7-96bd-f25fa56c6e69.JPG)

**Comment by anantgarg** [comment]
Yes, this is a known issue. I haven't had the time to resolve it. If someone can fix it and raise a pull request, I'd be happy to merge it.

**Comment by alexionegit** [comment]
Thanks! I can't use the app due to that. I would love to fix it, but don't know how... any hints please? 
I love the app's potential until then :)

Many thanks!
Alex

**Comment by anantgarg** [comment]
The Facebook API has been updated. So it needs to be updated in this code as well.

**Comment by alexionegit** [comment]
Hi ~ Could you please guide me on how to do this? I am willing to... I would like to use this powerful tool please. Many thanks!

**Comment by anantgarg** [comment]
You'll have to check the code to figure out the issue with the new API. I'd suggest getting a dev on Upwork to help you out. Should be less than 30 minutes of work.
